circle.yml

```diff
@@ -0,0 +1,35 @@
+# CircleCI Configuration file
+
+machine:
+  python:
+      version: 2.7.11
+
+general:
+  artifacts:
+     - "dist"
+     - "cover"
+
+dependencies:
+  override:
+    - pip install tox tox-pyenv
+    - pyenv local 2.7.11 3.5.1  # Should correspond to pre-installed Python versions on CircleCI
```

Comment:
Only `3.6.x`

Reply:
This is going away when all of our builds are united under globality-build (coming in the next couple of days)